Electrify.SmartMeterUi/Services/UsageService.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Electrify.Models.Models;
+using Electrify.SmartMeterUi.Services.Abstractions;
+
+namespace Electrify.SmartMeterUi.Services;
+
+internal class UsageService : IUsageService
```

Comment:
Instead of having start stop isnt it better to just create the timer in the constructor with the callback set so it just starts immediately and then implement IDisposable which disposes of the timer? I would also advocate for using [System.Threading.Timer](https://learn.microsoft.com/en-us/dotnet/api/system.threading.timer?view=net-8.0) not System.Timers.Timer as it asynchronous.